### PR TITLE
Allow L515 trigger/reset CAH from terminal [Viewer]

### PR DESCRIPTION
**Pull request by Nir-Az**
The terminal commands now support to manual features.
Command >  "trigger-camera-accuracy-health"
Command >  "reset-camera-accuracy-health"

This commands will pop up the related GUI modal is supported.

Currently the command will affect the first device that support the feature from all the connected devices.

### CODE REVIEW

**Comment by Nir-Az** [comment]
Can be tracked on [RS5-8462]

File: common/model-views.cpp [hunk 1]
```diff
@@ -3379,9 +3379,14 @@ namespace rs2
         , _detected_objects(std::make_shared< atomic_objects_in_frame >()),
         _updates(viewer.updates)
     {
-        if (dev.is<device_calibration>())
+        if( dev.supports( RS2_CAMERA_INFO_PRODUCT_LINE )
```
Comment (maloel) [inline common/model-views.cpp]: Why? Something was wrong with the device_calibration?

File: common/model-views.h [hunk 2]
```diff
@@ -777,6 +777,7 @@ namespace rs2
 
 
         std::shared_ptr< atomic_objects_in_frame > get_detected_objects() const { return _detected_objects; }
+        bool is_cah_model_enabled() { return _accuracy_health_model ? true : false; };
```
Comment (maloel) [inline common/model-views.h]: const

Comment (maloel) [reply common/model-views.h]: And you have an extra ; at the end

File: common/model-views.cpp (hunk 1 above)
Comment (Nir-Az) [reply common/model-views.cpp]: It's still there, view the entire file. the order is a bit changed and formatted differently, not shown here.

File: common/model-views.h (hunk 2 above)
Comment (Nir-Az) [reply common/model-views.h]: Correct, will be fixed

**Review by maloel — APPROVED** [review]

**Pull request merged**